openstudiocore/src/utilities/filetypes/EpwFile.cpp

```diff
@@ -132,67 +132,119 @@ EpwDataPoint::EpwDataPoint(int year,int month,int day,int hour,int minute,
   setLiquidPrecipitationQuantity(liquidPrecipitationQuantity);
 }
 
-boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(std::string line)
+boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(const std::string &line)
+{
+  std::vector<std::string> list;
+  boost::split(list, line, boost::is_any_of(","));

```

Comment:
@kbenne that is one of what I googled... and sent to you.

Also I agree with @kbenne about not needing either except in a few instances.